src/scripts/common.ts

```diff
@@ -269,6 +269,18 @@ namespace Scripts {
             }
         }
 
+        static poisonLastAttack() {
+            const kitSerial = Scripts.Utils.findFirstType(gameObject.uncategorized.apprenticesPoisoningKit);
+            if (kitSerial) {
+                Orion.WarMode(false);
```

Comment:
ono by melo stacit dat jen `Orion.WarMode(true)` a pak mozna nejakou lehkou pauzu aby to server zpracoval... prepinani warmodu pres false a pak true neni potreba.. vyzkousej si to kdyztak